slam-ws/src/igvc_world/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(usma_gazebo)
+
+## Find catkin macros and libraries
+## if COMPONENTS list like find_package(catkin REQUIRED COMPONENTS xyz)
+## is used, also find other catkin packages
+find_package(catkin REQUIRED)
```

Comment:
Add new line

Reply:
You're right, that's unclear. I did not mean for those two to be concurrent steps, more to be like two different things you can do after the first two steps are done. I will reword it